src/SumoCoders/FrameworkCoreBundle/Resources/assets/coffee/framework.coffee

```diff
@@ -142,19 +142,27 @@ class Framework extends DefaultObject
     )
 
   _initSortable: ->
-    $( '.sortable' ).sortable()
-    handle: 'button'
-    cancel: ''
+    return if (!$.isFunction($.fn.sortable))
```

Comment:
Op de login pagina is jQuery UI niet geladen, dus daar krijgen we die error (net als de UI functies + select2, die ook niet ingeladen is). Maar ik kan wel eerst checken of er elementen zijn met sortable class.